### PR TITLE
Create EnergyConsumption

### PR DESCRIPTION
**Pull request by rgopikrishnan91**

### CODE REVIEW

**Comment by rgopikrishnan91** [comment]
@kestewart @goneall Can you guys please review this its part 1 of the changes. Other PRs to follow

File: model/AI/Classes/EnergyConsumption [hunk 1]
```diff
@@ -0,0 +1,26 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# EnergyConsumption
+
+## Summary
+
+Class that contains properties to describe EnergyConsumption incurred by a AI model in different stages of its lifecycle.
```
Comment (bact) [inline model/AI/Classes/EnergyConsumption:7]: "to describe EnergyConsumption" -> "to describe energy consumption"

File: model/AI/Classes/EnergyConsumption [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+## Description
+
+The class used for denoting the training energy consumption, inference energy consumption and fine tuning energy consumption of the AI model(s) used in a AI software.
```
Comment (bact) [inline model/AI/Classes/EnergyConsumption:11]: Using "system" instead of "software"?

"used in a AI software" -> "used in an AI system"

**Review by kestewart — CHANGES_REQUESTED** [review]
Suggest applying the changes Art proposes,  and merge into 1 PR with the other PRs.

Please also add your signed off on the revised PR.

**Comment by rgopikrishnan91** [comment]
Closing PR since this is now part of #697

**Pull request closed without merging**